tests/python/release_checklist/check_lerobot_dataloader.py

```diff
@@ -2,37 +2,34 @@
 
 import os
 from argparse import Namespace
-from uuid import uuid4
 
 import rerun as rr
 
 README = """\
 # LeRobot dataloader check
 
-This will load an entire LeRobot dataset -- simply make sure that it does 🙃
+This will load a small LeRobot dataset -- simply make sure that it does.
 
-The LeRobot dataset loader works by creating a new _recording_ (⚠)️ for each episode in the dataset.
-I.e., you should see a bunch of recordings below this readme (3, to be exact).
+The LeRobot dataset loader works by creating a new _recording_ for each episode in the dataset.
+I.e., you should see exactly 3 recordings, corresponding to episode 0, 1 and 2.
 """
 
 
-def log_readme() -> None:
-    rr.log("readme", rr.TextDocument(README, media_type=rr.MediaType.MARKDOWN), static=True)
-
-
 def run(args: Namespace) -> None:
-    rr.script_setup(args, f"{os.path.basename(__file__)}", recording_id=uuid4())
+    rec = rr.script_setup(args, f"{os.path.basename(__file__)}", "episode_0")
```

Comment:
explicit is better than implicit

```suggestion
    rec = rr.script_setup(args, application_id=f"{os.path.basename(__file__)}", recording_id="episode_0")
```